.travis.yml

```diff
@@ -3,19 +3,19 @@ python:
 - '2.7'
 sudo: false
 env:
-- DB=sqlite3
-- DB=postgres
+- DATABASE_URL=sqlite:///db.sqlite3
+- DATABASE_URL=postgres://postgres@localhost:5432/brambling_test
```

Comment:
TBH I would be fine with us only testing against postgres, since this is an application and not a reusable app, and we should just always be using postgres.